src/Tribe/Editor.php

```diff
@@ -59,17 +59,64 @@ public function hook() {
 		add_filter( 'use_block_editor_for_post_type', [ $this, 'deactivate_blocks_editor_venue' ], 10, 2 );
 		add_filter( 'use_block_editor_for_post_type', [ $this, 'deactivate_blocks_editor_organizer' ], 10, 2 );
 		add_filter( 'use_block_editor_for_post_type', [ $this, 'deactivate_blocks_editor_event' ], 10, 2 );
+
+		add_filter( 'get_block_templates', [ $this, 'include_archive_events' ], 25, 3 );
+	}
+
+	/**
+	 * Modify the available Templates so that people can edit the template.
+	 *
+	 * @since TBD
+	 *
+	 * @param array     $query_result
+	 * @param \WP_Query $query
+	 * @param string    $template_type Which kind of template we are talking about.
+	 *
+	 * @return array
+	 */
+	public function include_archive_events( $query_result, $query, $template_type ) {
+		$template_slug = 'archive-' . Tribe__Events__Main::POSTTYPE;
+
+		if ( 'wp_template' !== $template_type ) {
+			return $query_result;
+		}
+
+		// If we are not querying for all or the specific one we want we bail.
+		if ( ! empty( $query['slug__in'] ) && ! in_array( $template_slug, $query['slug__in'], true ) ) {
+			return $query_result;
+		}
+
+		$template                 = new WP_Block_Template();
+		$template->id             = 'the-events-calendar//archive-events';
+		$template->theme          = 'The Events Calendar';
+		$template->content        = _inject_theme_attribute_in_block_template_content( '
+<!-- wp:template-part {"slug":"header","tagName":"header"} /-->
+<!-- wp:tribe/archive-events {} /-->
+<!-- wp:template-part {"slug":"footer","tagName":"footer"} /-->
+		' );
+		$template->slug           = $template_slug;
+		$template->source         = 'theme';
+		$template->theme          = wp_get_theme()->get_stylesheet();
+		$template->type           = $template_type;
+		$template->title          = esc_html__( 'Events Archive', 'the-events-calendar' );
+		$template->status         = 'publish';
+		$template->has_theme_file = true;
+		$template->is_custom      = true;
+
+		$query_result[] = $template;
+
+		return $query_result;
 	}
 
 	/**
 	 * For now we don't use Blocks editor on the Post Type for Organizers
 	 *
-	 * @todo  see https://core.trac.wordpress.org/ticket/45275
+	 * @todo   see https://core.trac.wordpress.org/ticket/45275
```

Comment:
Restore the removed white space.

```suggestion
	 * @todo  see https://core.trac.wordpress.org/ticket/45275
```